internal/veracode-api-wrapper/updatemitigationinfo/README.md

```diff
@@ -0,0 +1,48 @@
+**Veracode API Wrapper - Update Mitigation Info**
+
+Calls the Veracode api to update the mitigation comment of a build id. This is effectively used in updating the comment 
+in veracode which is maintained in CSV files.
+
+**Prerequisite**
+
+Prerequesties specific to Veracode :  Veracode Subcription , User role with "Mitigation" and "Mitigation and Comments"
+Prerequesties specif to program    :  <a href="https://github.com/wso2/security-tools/blob/master/internal/veracode-api-wrapper/updatemitigationinfo/requirements.txt">requirements.txt</a>
+
+**Local Installation**
+
+`$ virtualenv veracodeApiWrapper<br/>`
+
+`$ . env/bin/activate<br/>`
+
+`(veracodeApiWrapper) $ cd <Path to veracode-api-wrapper/updatemitigationinfo/scripts><br/>
+`
+
+``(veracodeApiWrapper) $ python updatemitigationinfo.py -h<br/>`
+``
+
+**Help**
+
+````
```

Comment:
Use 3 back-ticks here. Using 4 is not necessary. Same applies to the end of this block.